src/stores/inactive-users.ts

```diff
@@ -0,0 +1,49 @@
+import { sleep } from './../services/utils'
+import { defineStore } from 'pinia'
+import { computed, ref, watch } from 'vue'
+import { User } from '../pages/users/types'
```

Comment:
I still don't understand why you would need 2 stores.

If you want to make store that is heavily coupled to users page - then one store makes more sense (and you can just keep filter there).